src/components/explore/tuits-by-tag.js

```diff
@@ -75,45 +42,9 @@ const TuitsByTag = () => {
 
     // When we first load the page...
     useEffect(() => {
-        //setTuits([]);// Show no tuits...
         findTuitsWithTag();
-        /*
-        console.log("Use Effect activated");
-        if(sortOrder === 'LIKES'){
-            const tuitOrder = tuits.sort((a, b) => a.stats.likes - b.stats.likes);
-            setTuits(tuitOrder);
-            console.log("likes order");
-        }
-        //goToSearch();
-        Note: this can later be changed to show all tuits with any tag.
-        This can later be discussed by the team how we want to present this
-         */
     }, []);
 
-
-    // Whenever a keyboard button is clicked...
-    const handleKeyDown = (event) => {
```

Comment:
I removed it because it was grayed-out in IntelliJ with a statement saying it was unused. You might be able to see the same message if you check out main and hover over the function name.

Reply:
Makes sense!